src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -15,25 +15,41 @@
 use App\Entity\Submit;
 use App\Entity\Talk;
 use App\Repository\ConferenceRepository;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 class SlackNotifier
 {
+    private bool $hasNewConferences = true;
+    private bool $hasEndingCfps = true;
```

Comment:
This class is a service. I must not have a state.
You should remove theses variables.

More over, you are computing too much things `buildConferencesBlocks` and `buildCfpsBlocks` in case there is nothing to display. The code could be simplified